distribution/docs/src/main/resources/_contents/_transformers/developing-metacard-transformers-contents.adoc

```diff
@@ -0,0 +1,56 @@
+
+=== Developing Metacard Transformers
+
+In general, a `MetacardTransformer` is used to transform a `Metacard` into some desired format useful to the end user or as input to another process.
+Programmatically, a `MetacardTransformer` transforms a `Metacard` into a `BinaryContent` instance, which contains the translated `Metacard` into the desired final format.
+Metacard transformers can be used through the Catalog Framework `transform` convenience method or requested from the OSGi Service Registry by endpoints or other bundles.
+
+[discrete]
+==== Creating a New Metacard Transformer
+
+Existing input transformers are written as Java classes, and these steps walk through the steps to create a custom input transformer.
+
+. Create a new Java class that implements `${ddf-branding}.catalog.transform.MetacardTransformer`. +
+`public class SampleMetacardTransformer implements ${ddf-branding}.catalog.transform.MetacardTransformer`
+. Implement the `transform` method. +
```

Comment:
May not be the right place, but this came up recently. `transform` _must_ return a `Metacard` or throw an exception. It _cannot_ return null.